composer.json

```diff
@@ -15,11 +15,11 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "league/commonmark": "dev-master"
```

Comment:
@GrahamCampbell, will you change it to `0.4.*` instead just to make the current state stable, so?